### DISCUSSION TITLE
What is your favorite anime?

### DISCUSSION THREAD

**Discussion by junker530** [NAIST-SE/AttractingContributors#1152] [2024 Attraction Contributions]
https://github.com/junker530/Shota-Repo

**Comment by ryojiagatsuma1004** [comment]
Hello, I contributed to your repo!
Would you mind adding something to mine as well?
Here's my repo: https://github.com/ryojiagatsuma1004/AttractingContributors-Color

**Comment by kohanaist** [comment]
I have made a PR to your repo.
Could you contributed to my repo?
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
